src/__tests__/pkgTypes.test.ts

```diff
@@ -0,0 +1,97 @@
+import type { File } from '../jsDelivr';
+import * as api from '../pkgTypes';
+
+describe('package module/style types', () => {
+  describe('package style types', () => {
+    it('should return correct style types for multiple packages', () => {
+      const styleTypes = api.getStyleTypesForAll(
+        [
+          { styleTypes: [] },
+          { styleTypes: [] },
+          { styleTypes: [] },
+          { styleTypes: [] },
+          { styleTypes: ['css'] },
+        ],
+        [
+          [],
+          [{ name: '/dist/style/style.min.css' }],
+          [
+            { name: '/src/style/style.less' },
+            { name: '/dist/style/style.min.css' },
+            { name: '/dist/js/lib.min.js' },
+            { name: '/style.scss' },
+          ],
+          undefined,
+          [{ name: '/src/style/style.less' }],
+        ] as File[][]
```

Comment:
Usually not a fan of using `as Type`, this is handy for testing but usually makes the job of refactoring harder because TS will not complain if something is missing. 
What I favor is using a base object that I can spread and reuse, it decrease a bit the readability but will help future me/you.

Reply:
agreed, usually something like `createFile(partial)` should deal with the default values needed